bcipy/helpers/tests/test_load.py

```diff
@@ -161,5 +163,45 @@ def test_load_experiment_fields_raises_invalid_experiment_on_incorrectly_formatt
             load_experiment_fields(invalid_experiment_field_name)
 
 
+class TestUserLoad(unittest.TestCase):
+
+    def setUp(self):
+        # setup parameters to pass to load users method, it expects a key of data_save_loc only
+        self.directory_name = 'test_data_load_user'
+        self.parameters = {
+            'data_save_loc': f'{self.directory_name}/'
+        }
+
+    def test_user_load_with_no_directory(self):
+        response = load_users(self.parameters)
+
+        self.assertEqual(response, [])
+
+    # def test_user_load_no_data_save_location_provided_parameters(self):
```

Comment:
If changing the way it passes parameters as I mention above, you could then set self.directory == `f'{self.directory_name}/'`, then pass it to load_users.